src/Xrefcheck/Core.hs

```diff
@@ -38,6 +38,14 @@ data RepoType
   | GitLab
   deriving (Show)
 
+allRepoTypes :: [RepoType]
+allRepoTypes = [GitHub, GitLab]
```

Comment:
Btw, you could also use `Bounded` and `Enum` I suppose.

Reply:
Yeah, I wanted to do so first, but then imagined that there can be some GitHub-specific settings in the future (some field in `GitHub` ctor), so preferred to go with this thing from the beginning.